lib/iris/_data_manager.py

```diff
@@ -263,23 +297,6 @@ def _realised_dtype_setter(self, realised_dtype):
                 # Check the manager contract, as the managed dtype has changed.
                 self._assert_axioms()
 
-    def core_data(self):
-        """
-        If real data is being managed, then return the :class:`~numpy.ndarray`
-        or :class:`numpy.ma.core.MaskedArray`. Otherwise, return the lazy
-        :class:`~dask.array.core.Array`.
-
-        Returns:
-            The real or lazy data.
-
-        """
-        if self.has_lazy_data():
-            result = self._lazy_array
-        else:
-            result = self._real_array
-
-        return result
-
```

Comment:
I've relocated this to maintain the order of definitions within the class (this previously was a `@property` and promoted to a function)